### PR TITLE
Fix failing tests on CI

### PR DESCRIPTION
**Pull request by angrybayblade**
## Proposed changes

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request.

## Fixes

If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce to agents-aea?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply._

- [ ] I have read the [CONTRIBUTING](../CONTRIBUTING.md) doc
- [ ] I am making a pull request against the `develop` branch (left side). Also you should start your branch off our `develop`.
- [ ] Lint and unit tests pass locally with my changes and CI passes too
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that code coverage does not decrease.
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

DELETE INCLUSIVE THIS AND BELOW FOR STANDARD PR
------

## Release summary

Version number: [e.g. 1.0.1]

## Release details

Describe in short the main changes with the new release.

## Checklist

_Put an `x` in the boxes that apply._

- [ ] I have read the [CONTRIBUTING](../CONTRIBUTING.md) doc
- [ ] I am making a pull request against the `main` branch (left side), from `develop`
- [ ] Lint and unit tests pass locally and in CI
- [ ] I have checked the fingerprint hashes are correct by running (`scripts/generate_ipfs_hashes.py`)
- [ ] I have regenerated the latest API docs
- [ ] I built the documentation and updated it with the latest changes
- [ ] I have added an item in `HISTORY.md` for this release
- [ ] I bumped the version number in the `aea/__version__.py` file.
- [ ] I bumped the version number in every Docker image of the repo and published it. Also, I built and published them with tag `latest`  
      (check the READMEs of [`aea-develop`](../develop-image/README.md#publish) 
      and [`aea-user`](../user-image/README.md#publish))
- [ ] I have pushed the latest packages to the registry.
- [ ] I have uploaded the latest `aea` to PyPI.
- [ ] I have uploaded the latest plugins to PyPI.

## Further comments

Write here any other comment about the release, if any.

### CODE REVIEW

File: tests/test_decision_maker/test_default.py [hunk 1]
```diff
@@ -111,7 +111,7 @@ def setup(
             default_address_key=FetchAICrypto.identifier,
         )
         cls.config = {}
-        cls.decision_maker_handler = decision_maker_handler_cls(
```
Comment (DavidMinarsch) [inline tests/test_decision_maker/test_default.py:114]: why the change? Now it's not overridable from kwargs

File: tests/test_docs/test_agent_vs_aea/test_agent_vs_aea.py [hunk 2]
```diff
@@ -49,7 +49,7 @@ def setup_class(cls):
     def test_read_md_file(self):
         """Test the last code block, that is the full listing of the demo from the Markdown."""
         assert (
-            self.code_blocks[-1] == self.python_file
+            self.code_blocks[-1] == self.python_file[1:]
```
Comment (DavidMinarsch) [inline tests/test_docs/test_agent_vs_aea/test_agent_vs_aea.py:52]: I think basically in the docs you need to update the copyright string to match; and leave it here as it was

File: tests/test_docs/test_multiplexer_standalone/test_multiplexer_standalone.py [hunk 3]
```diff
@@ -48,7 +48,7 @@ def setup_class(cls):
     def test_read_md_file(self):
         """Read the code blocks. Last block should be the whole code."""
         assert (
-            self.code_blocks[-1] == self.python_file
+            self.code_blocks[-1] == self.python_file[1:]
```
Comment (DavidMinarsch) [inline tests/test_docs/test_multiplexer_standalone/test_multiplexer_standalone.py:51]: See above

File: tests/test_docs/test_standalone_transaction/test_standalone_transaction.py [hunk 4]
```diff
@@ -68,7 +68,7 @@ def setup_class(cls):
     def test_read_md_file(self):
         """Test the last code block, that is the full listing of the demo from the Markdown."""
         assert (
-            self.code_blocks[-1] == self.python_file
+            self.code_blocks[-1] == self.python_file[1:]
```
Comment (DavidMinarsch) [inline tests/test_docs/test_standalone_transaction/test_standalone_transaction.py:71]: see above

**Review by DavidMinarsch — CHANGES_REQUESTED** [review]
For the failing cosmwasm tests please add a pytest skip marker and then open an issue. Let's wait till they update the underlying pycosm library.

File: tests/test_docs/test_agent_vs_aea/test_agent_vs_aea.py (hunk 2 above)
Comment (angrybayblade) [reply tests/test_docs/test_agent_vs_aea/test_agent_vs_aea.py:52]: I changed these to see if it fixed the problem 😅

**Review by DavidMinarsch — APPROVED** [review]
LGTM
